apps/rush-lib/src/data/PackageReviewConfiguration.ts

```diff
@@ -58,120 +46,77 @@ export class PackageReviewItem {
  * This represents the JSON file specified via the "packageReviewFile" option in rush.json.
  * @public
  */
-export default class PackageReviewConfiguration {
+export class PackageReviewConfiguration {
```

Comment:
>PackageReviewConfiguration [](start = 13, length = 26)

This needs to get renamed PackageReviewConfiguration --> ApprovedPackagesConfiguration, but I will do that in a separate PR.  (When I tried it in this PR, the diff got seriously confusing.)

Same for all the other places that say "PackageReview"